benchmark/src/main/resources/benchmark.xsd

```diff
@@ -443,6 +443,9 @@
           <xs:element minOccurs="0" name="constraintStreamImplType" type="tns:constraintStreamImplType"/>
                               
           
+          <xs:element minOccurs="0" name="constraintStreamShareLambdas" type="xs:boolean"/>
```

Comment:
Not sure about the name. Maybe we should avoid implementation details?

- `constraintStreamAutomaticNodeSharing` (true/false)
- `constraintStreamNodeSharing` (auto/none)

Reply:
I prefer `constraintStreamAutomaticNodeSharing`, since technically, node sharing can be done (and will be done) already, just in a way that very annoying for the user.